categories.xml

```diff
@@ -1,4 +1,5 @@
 <categories>
-	<category name="Uncategorized" slug="uncategorized"/>
+	<category name="Uncategorized" slug="uncategorized"></category>
+        <category name="Configuration and Utilities" slug="config"></category>
```

Comment:
@raphamorim, which content are you going to be put into `Configuration and Utilities`? I think it will be clear to figure the naming after knowing that first.